ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -24,12 +35,35 @@ const headMarkerData = {
   heartStackMarker: '0205',
 } as const;
 
+const poisonOutputStrings = {
+  defamationOnYou: {
+    en: 'Defamation on YOU',
+    de: 'Ehrenstrafe aud DIR',
+    fr: 'Diffamation sur VOUS',
+    ja: '自分の巨大な爆発',
```

Comment:
[MAY] It would be more natural and easier to understand in Japanese to write `自分に巨大な爆発` (`jibun ni kyodai na bakuhatsu`) rather than `自分の巨大な爆発` (`jibun no kyodai na bakuhatsu`).

```suggestion
    ja: '自分に巨大な爆発',
```

Reply:
> [MAY] It would be more natural and easier to understand in Japanese to write `自分に巨大な爆発` (`jibun ni kyodai na bakuhatsu`) rather than `自分の巨大な爆発` (`jibun no kyodai na bakuhatsu`).

I copied this from the defamation in P9S - should it be updated there too?  Or is this change something more unique to this particular mechanic?

Reply:
> > [MAY] It would be more natural and easier to understand in Japanese to write `自分に巨大な爆発` (`jibun ni kyodai na bakuhatsu`) rather than `自分の巨大な爆発` (`jibun no kyodai na bakuhatsu`).
> 
> I copied this from the defamation in P9S - should it be updated there too? Or is this change something more unique to this particular mechanic?

I see. This isn't unique to this particular mechanic. Rather, I think we should modify 'the defamation' in P95 similarly to use 'jibun ni' (自分に) to keep the expression consistent across the board. Would you like me to create a small PR for that change?

Reply:
Is this common to all Defamation-style mechanics? [Alexander 12 Savage features a similar mechanic](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/03-hw/raid/a12s.ts#L206), and so does[ Alexander Ultimate](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.ts#L2017), in [two separate places](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.ts#L2157). (There are other places within the codebase, but these are just the first ones that came to mind.) If it's something that should be changed across the board like that, it might be best to add this to `outputs.ts` and then manually update the combination mechanics like the second iteration I linked on Alexander Ultimate.

Reply:
> Is this common to all Defamation-style mechanics? [Alexander 12 Savage features a similar mechanic](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/03-hw/raid/a12s.ts#L206), and so does[ Alexander Ultimate](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.ts#L2017), in [two separate places](https://github.com/OverlayPlugin/cactbot/blob/70eb959f8a1a907a886743f3d1e6577cb31668f7/ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.ts#L2157). (There are other places within the codebase, but these are just the first ones that came to mind.) If it's something that should be changed across the board like that, it might be best to add this to `outputs.ts` and then manually update the combination mechanics like the second iteration I linked on Alexander Ultimate.

I completely agree with your suggestion. This mechanic appears quite frequently across various battle contents in FF14, so managing it centrally in `outputs.ts` would indeed be an effective and efficient approach.

Reply:
I'll update the PR to drop this in `outputs.ts`, and I'll separately PR the change for the other places its used (so as not to clutter this PR).